msteams-platform/resources/create-messaging-extension-using-appstudio.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Create a messaging extension using App Studio
+author: clearab
+description: Learn how to create a Microsoft Teams messaging extension using App Studio.
+ms.topic: conceptual
+ms.author: anclear
+---
+# Create a messaging extension using App Studio
```

Comment:
Why is this showing as newly added page?

Reply:
As this particular file was deprecated recently, created this file with the same content and added under resources in TOC, as still developers refer to this file to create message extension using App Studio.